pyproject.toml

```diff
@@ -108,9 +108,9 @@ filterwarnings = [
     'ignore: `photutils.DAOStarFinder` is a deprecated alias:DeprecationWarning',
     'ignore: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.:UserWarning',
     # Ignore warning from astropy.io
-    'ignore: Card is too long, comment will be truncated.',
+    'ignore: Card is too long, comment will be truncated.:VerifyWarning',
```

Comment:
Ah, sorry, `VerifyWarning` is not a built-in warning, so the tests are failing....  remove.